apps/coordinator/src/components/ScriptExplorer/UTXOSet.jsx

```diff
@@ -173,11 +174,32 @@ class UTXOSet extends React.Component {
   renderInputs = () => {
     const { network, showSelection, finalizedOutputs } = this.props;
     const { localInputs } = this.state;
+    const dustLimits = this.props.dustLimits;
+
+    function getChipProperties(amountSats) {
+      const lowerLimit = dustLimits.lowerLimit;
+      const upperLimit = dustLimits.upperLimit;
+      let chipColor;
+      let chipLabel;
+      if (amountSats <= lowerLimit) {
+        chipColor = "#f55656"; // Indicating critical state
```

Comment:
Please use the MUI theme for defining colors so these can be updated programmatically and also fit with the rest of the theme: https://mui.com/material-ui/customization/palette/#default-colors

probably using `success`, `warning`, and `error`